lib/pool.js

```diff
@@ -52,6 +63,8 @@ class Pool {
 
```

Comment:
```js
const onConnect = function () {
  pool.emit('connect', this)
}
```

This way you only create one closure, instead of one for each client.

Reply:
Yes, you're right. Thank you!